content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md

```diff
@@ -299,7 +299,7 @@ The simplest way to get involved with Kubernetes is by joining one of the many [
 - 在 Twitter 上关注我们 [@Kubernetesio](https://twitter.com/kubernetesio)，了解最新动态
 - 在 [Discuss](https://discuss.kubernetes.io/) 上参与社区讨论 
 - 加入 [Slack](http://slack.k8s.io/) 上的社区
```

Comment:
```suggestion
- 加入 [Slack](https://slack.k8s.io/) 上的社区
```

---

FYI: As-is: https://raw.githubusercontent.com/kubernetes/website/main/content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md